src/Common/ThreadFuzzer.cpp

```diff
@@ -22,7 +22,7 @@
 
 
 /// We will also wrap some thread synchronization functions to inject sleep/migration before or after.
-#if defined(OS_LINUX) && !defined(THREAD_SANITIZER) && !defined(MEMORY_SANITIZER)
+#if defined(OS_LINUX) && !defined(THREAD_SANITIZER) && !defined(MEMORY_SANITIZER)  && !defined(__loongarch64)
```

Comment:
Why not?

Reply:
The `pthread_mutex_lock` function in the loongarch64 platform does not have an internal symbol similar to "__pthread_mutex_lock@{GLIBC_VERSION}." The loongarch64's format is:
```sh
root@clickhouse-cross /w/c/ClickHouse (loongarch64)# readelf -sW contrib/sysroot/linux-loongarch64/lib/loongarch64-linux-gnu/libc.so.6 | grep "pthread_mutex_lock"
  1265: 0000000000081578   828 FUNC    GLOBAL DEFAULT   13 pthread_mutex_lock@@GLIBC_2.36
```
This is different from x86-64, arm64, riscv64 etc.

Reply:
This modification is because I thought before that ThreadFuzzer could not intercept the "pthrad_mutex_lock" function without the "__pthrad_mutex_lock" symbol.

After referring to https://github.com/ClickHouse/ClickHouse/commit/bdeb5514c5e49b2f6158ca5b1f519956e55a03ec, I realized that ThreadFuzzer can intercept `pthrad_mutex_lock` through `MAKE_WRAPPER_USING_DLSYM`. So I will modify my code.